lib/apis/places.js

```diff
@@ -18,6 +18,38 @@
 var utils = require('../internal/convert');
 var v = require('../internal/validate');
 
+/**
+ * A Find Place request takes a text input, and returns a place.
+ * The text input can be any kind of Places data, for example,
+ * a name, address, or phone number.
+ *
+ * @memberof! GoogleMapsClient
+ * @name GoogleMapsClient.findPlaces
+ * @function
+ * @param {Object} query
+ * @param {string} query.input
+ * @param {string} query.inputtype
+ * @param {string} [query.language]
+ * @param {Array<string>} [query.fields]
+ * @param {ResponseCallback} callback Callback function for handling the result
+ * @return {RequestHandle}
+ */
+exports.findPlaces = {
+  url: 'https://maps.googleapis.com/maps/api/place/findplacefromtext/json',
+  validator: v.object({
+    input: v.string,
+    inputtype: v.oneOf(['textquery', 'phonenumber']),
+    language: v.optional(v.string),
+    fields: v.optional(utils.arrayOf(v.oneOf([
+      'alt_id', 'formatted_address', 'geometry', 'icon', 'id', 'name',
+      'permanently_closed', 'photos', 'place_id', 'scope', 'types',
+      'vicinity', 'opening_hours', 'price_level', 'rating'
+    ]), ',')),
+    retryOptions: v.optional(utils.retryOptions),
+    timeout: v.optional(v.number)
+  })
```

Comment:
we also support locationbias paramater https://developers.google.com/places/web-service/search#FindPlaceRequests

Reply:
Thanks, that's added now in bab91d76748b207f3614bed991d071af1d28fced